ACCEPTANCE_TESTING.md

```
@@ -107,8 +107,9 @@ That's it! No other steps needed.
 
 Once running, open your browser:
 
-- **Main UI**: http://localhost:8501
-- **API Docs**: http://localhost:8000/docs
+- **Main UI**: http://localhost:3000 (Docusaurus Docs & Results)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)